about.html

```diff
@@ -32,17 +32,25 @@
             </div>
         </div>
     </nav>
-    <section id="hero">
-        <div class="right-section">
+
+        <div class="about">
             <h1>About</h1>
-            <p class="subtitle">Typing Test is an open source project created by Arpit Ghura. The purpose of this community project is to collaborate with web developers from around the world. Our goal is to build a useful tool to practice and improve typing skills. If you would like to contribute to our project, click on the GitHub link at the top right of the page.</p>
+            <div class="subtitle">
+            <p> 
+            Typing Test is an open-source project created by <i>Arpit Ghura</i>.
+            The purpose of this community project is to collaborate with web developers from around the world.
+            We aim to build a valuable tool to practice and improve typing skills. Which will help you in further coding and in many gvt exams.<br>
```

Comment:
instead of `gvt` write the full form i.e., `government`

Reply:
ok sure